client/src/components/payments/checkoutForm.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from "react";
+import { pay } from '../../api/payments';
+
+import {
+  CardElement,
+  CardNumberElement,
+  CardExpiryElement,
+  CardCVCElement,
+  PaymentRequestButtonElement,
+  StripeProvider,
+  Elements,
+  injectStripe
+} from 'react-stripe-elements';
+
+import './checkoutForm.css'
+
+const handleBlur = () => {
+  console.log('[blur]');
+};
+const handleChange = (change) => {
+  console.log('[change]', change);
+};
+const handleClick = () => {
+  console.log('[click]');
+};
+const handleFocus = () => {
+  console.log('[focus]');
+};
+const handleReady = () => {
+  console.log('[ready]');
+};
+
+const createOptions = (fontSize, padding) => {
+  return {
+    style: {
+      base: {
+        fontSize,
+        color: '#424770',
+        letterSpacing: '0.025em',
+        fontFamily: 'Source Code Pro, monospace',
+        '::placeholder': {
+          color: '#aab7c4'
+        },
+        padding
+      },
+      invalid: {
+        color: '#9e2146'
+      }
+    }
+  };
+};
+
+class _CardForm extends React.Component {
```

Comment:
make this a functional component

Reply:
I removed it and added functionality to checkoutForm.